commons/nrjmx.thrift

```diff
@@ -0,0 +1,56 @@
+namespace java org.newrelic.nrjmx.v2.nrprotocol
+
+struct JMXConfig {
+  1: string connectionURL
+  2: string hostname,
+  3: i32 port,
+  4: string uriPath,
+  5: string username,
+  6: string password,
+  7: string keyStore,
+  8: string keyStorePassword,
+  9: string trustStore,
+  10: string trustStorePassword,
+  11: bool isRemote,
+  12: bool isJBossStandaloneMode
```

Comment:
would it make sense to have `isStandaloneMode` instead o jboss related one? os is it only jboss related?

Reply:
is only jboss related